src/electionguard/serializable.py

```diff
@@ -98,5 +112,10 @@ def set_deserializers() -> None:
     set_validator(lambda q: q.is_in_bounds(), ElementModQ)
 
     set_deserializer(
-        lambda usage_string, cls, **_: ProofUsage[usage_string], ProofUsage
+        lambda none, cls, **_: None
```

Comment:
This deserialization from "None" is critical. Perhaps we could change it to serialize to `null` as well and have a deserialization from `null` to None but regardless this needs to exist for Ballots to be deserialized.